src/client/eppo-client.spec.ts

```diff
@@ -150,7 +150,7 @@ describe('EppoClient E2E test', () => {
 
       const client = new EppoClient(storage);
       client.getStringAssignment(flagKey, 'subject-to-be-logged', {}, 'default-value');
-      client.setLogger(mockLogger);
+      client.setAssignmentLogger(mockLogger);
```

Comment:
Renamed to make it clear how it differs from the newly-added `setBanditLogger()`

Reply:
thanks, I was always confused by `setLogger`, always thought it was a logger in the generic meaning of the word in programming